go.mod

```diff
@@ -0,0 +1,51 @@
+module github.com/digitalocean/doctl
+
+go 1.12
+
+require (
+	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/cpuguy83/go-md2man v1.0.7 // indirect
+	github.com/digitalocean/godo v1.16.0
+	github.com/dustin/go-humanize v1.0.0
+	github.com/fatih/color v1.7.0
+	github.com/gobwas/glob v0.2.3
+	github.com/gogo/protobuf v1.1.1 // indirect
+	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
+	github.com/google/uuid v1.1.0 // indirect
+	github.com/imdario/mergo v0.3.6 // indirect
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/json-iterator/go v1.1.5 // indirect
+	github.com/mattn/go-colorable v0.0.9 // indirect
+	github.com/mattn/go-isatty v0.0.4 // indirect
+	github.com/mitchellh/ioprogress v0.0.0-20180201004757-6a23b12fa88e
+	github.com/mitchellh/mapstructure v1.1.2 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/natefinch/pie v0.0.0-20170715172608-9a0d72014007
+	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709
+	github.com/russross/blackfriday v1.5.2 // indirect
+	github.com/shiena/ansicolor v0.0.0-20151119151921-a422bbe96644
+	github.com/spf13/afero v1.2.2
+	github.com/spf13/cast v1.3.0 // indirect
+	github.com/spf13/cobra v0.0.3
+	github.com/spf13/jwalterweatherman v1.0.0
+	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/spf13/viper v1.2.1
+	github.com/stretchr/objx v0.1.1 // indirect
+	github.com/stretchr/testify v1.3.0
+	golang.org/x/crypto v0.0.0-20181203042331-505ab145d0a9
+	golang.org/x/net v0.0.0-20190206173232-65e2d4e15006 // indirect
+	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
+	golang.org/x/sys v0.0.0-20190312061237-fead79001313 // indirect
+	golang.org/x/text v0.3.1-0.20181227161524-e6919f6577db // indirect
+	golang.org/x/time v0.0.0-20181108054448-85acf8d2951c // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20190606204050-af9c91bd2759 // indirect
+	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
```

Comment:
Is this the right `k8s.io/apimachinery` as of `k8s.io/api@af9c91bd2759`? If I had to guess, I'd say that you upgraded `k8s.io/client-go` last, and it upgraded `k8s.io/api` on you, too. Kubernetes packages should generally be kept inline with each other.

Reply:
Ah, nice catch.